next/src/components/formatted-text.tsx

```diff
@@ -34,7 +34,7 @@ const options: HTMLReactParserOptions = {
         if (isRelative(src)) {
           return (
             <Image
-              src={`${env.NEXT_PUBLIC_DRUPAL_BASE_URL}${src}`}
+              src={`${env.DRUPAL_BASE_URL_INTERNAL}${src}`}
```

Comment:
I think here we should actually use `NEXT_PUBLIC_DRUPAL_BASE_URL`, or are we 100% sure this component will only ever load from the server?

In another project (pages router but otherwise very similar) where this line in FormattedText caused an error "Attempted to access a server-side environmental variable on the client", so had to change this one